src/lib/y2storage/disk_analyzer.rb

```diff
@@ -284,34 +295,39 @@ def release_name(filesystem)
       fs.release_name
     end
 
-    # TODO: there is no reason to initally discard usb devices. The behavior is
-    # kept to avoid big changes in release phase.
+    # Finds devices (disk devices or software raids) that are suitable for installing Linux
     #
-    # Finds disk devices that are suitable for installing Linux.
+    # From fate#326573 on, software raids with partition table or without children are also
+    # considered as valid candidates.
     #
-    # @note Usb devices are considered as candidate disk only if there are no
-    #   other candidate disks.
-    #
-    # @return [Array<BlkDevice>] candidate devices
+    # @return [Array<BlkDevice>] candidate devices (disk devices and/or software RAIDs matching the
```

Comment:
NP: again, I guess "/or" is leftover.

Reply:
Actually it is there intentionally, since it tried to say: "The returned candidate devices could be disk devices _and_ software RAIDs, _or_ only disk_devices _or_ only software RAIDs" .

However, if it is better expressed using only "and", I should change it :)

Reply:
Yes, I prefer to see "and" there, because I only want to know what kind of devices it could return, not the conditions in which it returns this or the other.